.github/actions/count-failed-runs/action.yaml

```diff
@@ -0,0 +1,65 @@
+name: "Count completed failed runs"
+description: This actions counts the number of consecutive failed runs for a given workflow.
+inputs:
```

Comment:
This is validated by this workflow run - https://github.com/radius-project/radius/actions/runs/8865909289/job/24342699250